src/riot-rs/src/lib.rs

```diff
@@ -35,9 +34,15 @@ pub use riot_rs_macros::task;
 #[cfg(any(feature = "threading", doc))]
 pub use riot_rs_macros::thread;
 
-// These are used by proc-macros we provide
-pub use linkme;
-pub use static_cell;
-
 // ensure this gets linked
 use riot_rs_boards as _;
+
+pub use riot_rs_embassy::api::*;
+pub use static_cell::{ConstStaticCell, StaticCell};
+
+pub mod crates {
```

Comment:
Nit: I find the name `crates` a bit confusing; imo it gives the impression of "these are riot-rs crates". Maybe call it `reexports` instead? 
Or how do other crates in the ecosystem call it?

Reply:
I chose "crates" over "reexports" b/c to me, a re-exported whole crate is more than a reexport of e.g., a single type or function. So to make clear that the module contains and only contains whole crates, I called it "crates".

I'm totally open to better suggestions, "reexports" is also possible.

Regarding other crates ... I guess the answer is "don't do that in the first place" ...

Reply:
meeting discussion: going for reexports